engine/apps/api/views/labels.py

```diff
@@ -51,26 +51,41 @@ class LabelsViewSet(LabelsFeatureFlagViewSet):
     def get_keys(self, request):
         """List of labels keys"""
         organization = self.request.auth.organization
-        result, response_info = LabelsAPIClient(organization.grafana_url, organization.api_token).get_keys()
-        return Response(result, status=response_info["status_code"])
+        try:
```

Comment:
looks like overriding the [`handle_exception`](https://www.django-rest-framework.org/api-guide/views/#handle_exceptionself-exc) method on the viewset would allow to keep things more DRY?

Reply:
makes sense